src/components/slop-fest-modal.jsx

```diff
@@ -210,7 +211,7 @@ export function SlopFestModal({ buttonTitle, location, fest }) {
             name={"invitees"}
           />
           <Form.Submit
-            title={buttonTitle}
+            title={"Fest on!"}
```

Comment:
We can see here that `buttonTitle` was actually a prop passed into the component on line 11. It's now also no longer used. In situations like this, it might best to find out where this component is loaded from, and instead pass in the `Fest on!` string from there.

Reply:
@marzahlde09 let's update this `title` here to reuse the original `buttonTitle` and change the prop we pass in to this component to keep it modularized. Thanks.

Reply:
I don't know why I wasn't seeing these notifications! Sorry for the delay. Handled =)